### PR TITLE
Onboarding: add cv advanced view and reset defaults functionality

### PR DESCRIPTION
**Pull request by 0xGabi**
Screen Shot 2021-09-05 at 2 22 16 AM
Screen Shot 2021-09-05 at 2 22 39 AM
Screen Shot 2021-09-05 at 2 22 44 AM

### CODE REVIEW

File: src/components/Onboarding/Screens/Apps/ConvictionVotingSettings/AdvancedSettingsModal.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    setRequestToken(requestTokenAddress)
+  }, [minThresholdStakePct, requestTokenAddress])
+
+  return (
+    <Modal
+      title="Advanced Settings"
+      visible={visible}
+      onClick={handleDone}
+      onClose={onClose}
+      width="100"
```
Comment (fabriziovigevani) [inline src/components/Onboarding/Screens/Apps/ConvictionVotingSettings/AdvancedSettingsModal.js:48]: Ah i think we need to add the unit here.
If we fix here we can remove the width set to the `Field`

```suggestion
      width="600px"
```

**Review by fabriziovigevani — COMMENTED** [review]
🙌 🙌 

eEft some suggestions and general corrections!

File: src/components/Onboarding/Screens/GardenMetadata.js [hunk 2]
```diff
@@ -212,18 +214,16 @@ function GardenMetadata() {
...
     } else {
       setDisplayErrors(true)
     }
   }, [errors, onConfigChange, onNext, formData])
 
-  const handleBack = useCallback(() => {
-    onConfigChange('garden', formData)
-    onBack()
-  }, [onConfigChange, onBack, formData])
-
```
[the remark this answers is not in the document]
Comment (0xGabi) [reply src/components/Onboarding/Screens/GardenMetadata.js:226]: I remember us having a conversation at some point that state should only be updated once we advance to a new step. And that's the behavior in the rest of the screens.

File: src/components/Onboarding/Screens/Apps/ConvictionVotingSettings/index.js [hunk 3]
```diff
@@ -87,6 +97,9 @@ function ConvictionVotingSettings() {
     stakeOnProposal: DEFAULT_STAKE_ON_PROPOSAL,
     stakeOnOtherProposals: DEFAULT_STAKE_ON_OTHER_PROPOSALS,
   })
+  const [openSettingsModal, setOpenSettingsModal] = useState(false)
```
[the remark this answers is not in the document]
Comment (0xGabi) [reply src/components/Onboarding/Screens/Apps/ConvictionVotingSettings/index.js:100]: Good point!

File: src/components/Onboarding/Screens/Apps/ConvictionVotingSettings/index.js [hunk 4]
```diff
@@ -87,6 +97,9 @@ function ConvictionVotingSettings() {
...
+
+  const { conviction: DEFAULT_CONVICTION_CONFIG } = DEFAULT_CONFIG
```
[the remark this answers is not in the document]
Comment (0xGabi) [reply src/components/Onboarding/Screens/Apps/ConvictionVotingSettings/index.js]: I tried but I got an error `ReferenceError: Cannot access 'DEFAULT_CONFIG' before initialization`. My understanding is that, when the module loads the global variables the `DEFAULT_CONFIG` is not yet defined.

**Pull request merged**